modules/oadp-plugins.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc
+
+[id="oadp-plugins_{context}"]
+= OADP plug-ins
+
+The OpenShift API for Data Protection (OADP) provides default Velero plug-ins that are integrated with storage providers to support backup and snapshot operations. You can create link:https://velero.io/docs/v1.7/custom-plugins/[custom plug-ins] based on the Velero plug-ins.
+
+OADP also provides plug-ins for {product-title} resource backups and Container Snapshot Interface (CSI) snapshots.
+
+[cols="3", options="header"]
+.OADP plug-ins
+|===
+|OADP plug-in |Function |Storage location
+
+.2+|`aws` |Backs up and restores Kubernetes objects by using object store. |AWS S3
+|Backs up and restores volumes by using snapshots. |AWS EBS
+
+.2+|`azure` |Backs up and restores Kubernetes objects by using object store. |Microsoft Azure Blob storage
+|Backs up and restores volumes by using snapshots. |Microsoft Azure Managed Disks
+
+.2+|`gcp` |Backs up and restores Kubernetes objects by using object store. |Google Cloud Storage
+|Backs up and restores volumes by using snapshots. |Google Compute Engine Disks
+
+|`openshift` |Backs up and restores {product-title} resources by using object store. ^[1]^ |Object store
+
+|`csi` |Backs up and restores volumes by using CSI snapshots. ^[2]^ |Cloud storage that supports CSI snapshots
+|===
+[.small]
+--
+1. Mandatory.
```

Comment:
I'm left wondering a few things. In what sense is this mandatory? Can't remove or turn off the plugin? Must be there for support reasons?

Reply:
They can turn it off but that would be pretty stupid because the backup wouldn't back up openshift resources. Ideally, it should be hardcoded somewhere so that the user can't remove it. However, this is a first release and some things are not nailed down.